managed-identity-function/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>asc-managed-identity-function-sample</artifactId>
+    <version>0.1.0</version>
+
+    <parent>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-starter-parent</artifactId>
+        <version>Greenwich.SR3</version>
```

Comment:
Could you update to Hoxton.SR8 and remove the reference to `spring-cloud-starter-azure-spring-cloud-client`?
It is not needed anymore.
You can refer to #33 .

Reply:
Thanks for the feedback. I've updated the references.

Reply:
Thanks for the quick follow up!